content/2021/04-09-token-exchange-optional-parameter-added.md

```diff
@@ -0,0 +1,32 @@
+---
+applied_at: '2021-04-09'
+applies_to:
+  - api
+is_impactful: false
+is_new_feature: true
+collapse: true
+show_excerpt: true
+release_source_url: ''
+---
+
+# Token Exchange Optional Parameter Added
+
+<!-- more -->
+
+```bash
+{
+  curl -i -X POST "https://api.box.com/oauth2/token" \
+     -H "Content-Type: application/x-www-form-urlencoded" \
+     -d "client_id=[CLIENT_ID]" \
+     -d "client_secret=[CLIENT_SECRET]" \
+     -d "code=[CODE]" \
+     -d "grant_type=authorization_code"\
+     -d "box_shared_link=https://cloud.box.com/s/123456" \
+}
+```
+
+## Updates
+
+When requesting a token, you now have the ability to downscope that token for a shared link using the link URL instead of using a `resource` URL. Password protected links are not supported. This option cannot be used in addition to the `resource` option nor can it be a shared link created on a weblink.
+
+- Added the ability to downscope a token for a [shared link](g://shared-links)
```

Comment:
Let's call out the specific updates that were made and cross link - recommend:

- "**Added new `box_shared_link` request parameter to downscoping documentation. A shared link may be supplied to downscope an access token in the following way.**". Crosslink "downscoping documentation" to https://developer.box.com/guides/authentication/access-tokens/downscope/#downscoping-in-practice and move the code sample at the top of the changelog below this item.
- "**Added `box_shared_link` request parameter to request access token API reference.**". Crosslink "request token API reference" to https://developer.box.com/reference/post-oauth2-token/#request.